persistence/src/main/scala/akka/persistence/kafka/journal/JournalAdapter.scala

```diff
@@ -68,7 +68,8 @@ object JournalAdapter {
         val clientId = clientIdOf(config.journal.kafka.producer.common)
         metrics(clientId)
       }
-      KafkaProducerOf[F](producerMetrics)
+      val prefix = clientIdOf(config.journal.kafka.consumer.common)
```

Comment:
client ID is already included in the generated labels

Reply:
in what kind of labels the ID already included?

Reply:
Please try to run it locally - client_id is already included in the labels generated by the collector.
I suspect if you generate the prefix this way, it won't even mention that those are "kafka producer" metrics.